tests/appsec/waf/test_blocking.py

```diff
@@ -196,6 +196,23 @@ def test_html_template_v2(self):
         assert self.r_html_v2.text == BLOCK_TEMPLATE_HTML_MIN_V2
 
 
+@rfc("https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2705464728/Blocking#Stripping-response-headers")
+@scenarios.appsec_blocking
+@features.appsec_blocking_action
+class Test_Blocking_strip_response_headers:
+    def setup_strip_response_headers(self):
+        self.r_srh = weblog.get(f"/tag_value/anything/200?content-language=krypton&x-secret-header=123")
```

Comment:
changing the order of headers so the custom header is set before the blocking header
```suggestion
        self.r_srh = weblog.get(f"/tag_value/anything/200?x-secret-header=123&content-language=krypton")
```